app/assets/stylesheets/_main.scss

```diff
@@ -328,7 +329,7 @@ $govuk-page-width: 1100px;
 }
 
 .single-section-accordion {
-  .govuk-accordion__controls {
+  .bops-govuk-accordion__controls {
```

Comment:
I would use `bops-accordion` and not `bops-govuk-accordion` so that search a second time doesn't pick up the substring match